src/main/java/uk/gov/hmcts/reform/notificationservice/service/NotificationMessageProcessor.java

```diff
@@ -59,6 +62,23 @@ public boolean processNextMessage() throws ServiceBusException, InterruptedExcep
         return message != null;
     }
 
+    private void handleDuplicateMessageId(IMessage message, String errorMessage)
+        throws InterruptedException, ServiceBusException {
+        if (message.getDeliveryCount() == 1) {
+            deadLetterTheMessage(
```

Comment:
still not sure it's wise to dead letter. it'll be same message from what we have experienced. with all this logic it would be like Schrödinger's cat: it is alive and dead at the same time

Reply:
Discussed in Slack. For the first delivery of the duplicate message Id, we dead letter the message. otherwise, we complete the message